### DISCUSSION TITLE
[V5] 调用clear()方法之后元素残留问题

### DISCUSSION THREAD

**Discussion by recursively** [antvis/G6#6330] [Q&A]
我目前使用clear()方法来试图重置画布，但是发现clear()并不能将前一次画布中的元素清除干净，如下所示：

<img width="1230" alt="image" src="https://github.com/user-attachments/assets/815655c6-ace4-4789-a264-722e4b677503">

每一次渲染都会带上前一次的元素，目前测试了tooltip, fisheye, minimap这几个插件，都存在这种问题，该如何解决呢？

**Comment by resetsix** [comment]
https://codesandbox.io/p/sandbox/22mp34

没复现出你说的元素残留，最好给一个**复现问题的最小在线示例**别人才能帮你定位出问题所在。

排查思路：
1、严格模式
2、组件卸载时是否清除上一次画布实例
